CatLover.Api/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	// "database/sql"
+	// _ "github.com/mattn/go-sqlite3"
+
+	"fmt"
+	"net/http"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+
+	"catlover.api/controllers"
+	"catlover.api/services"
+)
+
+func homePage(c *gin.Context) {
+	fmt.Println("Endpoint Hit: pageNotFound")
+	var response = services.ReturnResponse("404 Not Found", 404, "pageNotFound!", nil)
+	c.IndentedJSON(http.StatusOK, response)
+}
+
+func handleRequests() {
+
+	router := gin.Default()
+
+	router.Use(cors.New(cors.Config{
+		AllowMethods:  []string{"GET", "PATCH"},
+		ExposeHeaders: []string{"Content-Length"},
+		AllowHeaders:  []string{"*"},
+		AllowOrigins:  []string{"http://localhost:10000", "http://localhost:3000"},
```

Comment:
Really good you thought of CORS handling

Reply:
Thank you. We must always use CORS in the API in order to communicate secure with the client/SPA.